templates/quickstart-jira-dc.template.yaml

```diff
@@ -489,12 +492,16 @@ Parameters:
     ConstraintDescription: Must be the name of an existing Route53 Hosted Zone.
     Description: The domain name of the Route53 PRIVATE Hosted Zone in which to create cnames
     Type: String
+  JiraClusterNodeRole:
+    Default: ''
+    Description: The role used for AutoScalingGroup profile.
+    Type: String
+    ConstraintDescription: Must be a valid role. For required policies please see the resource JiraClusterNodeRoleDefault.
```

Comment:
I don't know if this description is helpful to users in the UI. They don't really see the resources. `JiraClusterNodeRoleDefault` will probably mean nothing to them. I think we need to make it clear it's optional and for advanced configuration only.

@ddomingorh what do you think

Reply:
Can you take a look at the new description?